### DISCUSSION TITLE
[Beta] Version 3.3

### DISCUSSION THREAD

**Discussion by vosmiic** [vosmiic/jellyfin-ani-sync#143] [Announcements]
Quite a few changes in this release, most notably anime detection improvements when dealing with media that has multiple season thanks to @tie.
The configuration area has been overhauled, with better error handling so users are able to clearly see when the configuration is incorrect or an error has occurred.
Also 429's are now being handled as users were reporting issues mainly when updating entities in AniList.
Creating a discussion for the release so do leave a message if you run into any issues.

## What's Changed
* Add missing "targetAbi" to new releases. by @TJ09 in https://github.com/vosmiic/jellyfin-ani-sync/pull/132
* Fixing Anilist timeout issues by @vosmiic in https://github.com/vosmiic/jellyfin-ani-sync/pull/131
* Frontend fixes by @vosmiic in https://github.com/vosmiic/jellyfin-ani-sync/pull/138
* partially fix episode offsets by @tie in https://github.com/vosmiic/jellyfin-ani-sync/pull/141

**Full Changelog**: https://github.com/vosmiic/jellyfin-ani-sync/compare/v3.2...v3.3b

<hr /><em>This discussion was created from the release <a href='https://github.com/vosmiic/jellyfin-ani-sync/releases/tag/v3.3b'>[Beta] Version 3.3</a>.</em>

**Comment by Arlind-dev** [comment]
Works fine for me, the new frontend is also pretty good.
Though none of my broken Anime got fixed by the "partially fix episode offsets" PR.

**Reply by tie** [reply]
@Arlind-dev, how do you manage metadata for your anime library?

In my case, Sonarr writes TVDB IDs in .nfo files that get picked up by Jellyfin. Anisync then uses offline anime database to map TVDB to AniDB ID. Then the corresponding MAL ID is fetched from [ARM server API](https://arm.haglund.dev/docs) that is used to update list entry on Shikimori (that has anime database mostly mirrored from MAL and allows querying titles by MAL ID).

Note that TVDB to AniDB is a one-to-many relation, that’s what the PR fixes (to a certain extent).

**Reply by Arlind-dev** [reply]
@tie, Hi very similar setup, I use Sonarr and let Jellyfin get the Metadata on its own (usually don't have a problem with this setup).

What I meant with "none of my broken Anime got fixed" is the anime that I described in https://github.com/vosmiic/jellyfin-ani-sync/issues/128 (ex. Naruto, Bleach and One Piece) still don't work yet with TheTVDB as my Metadata Source.
